scripts/md2html/build.sh

```
@@ -2,32 +2,86 @@
+    destination="$deploydir/$version.html"
+    maintainers="EDITORS.md"
+  else
+    destination="$deploydir/v$version.html"
+    maintainers="$(dirname $specification)/$version-editors.md"
+  fi
+
+  minorVersion=${version:0:3}
+  tempfile="$deploydir/temp/$version.html"
+  tempfile2="$deploydir/temp/$version-2.html"
+
+  echo === Building $version to $destination
```

Shellcheck (0.10.0)

[error] Use [ x = y ] to compare values (or quote '==' if literal).

(SC2284)